policyengine_us/parameters/gov/states/ct/tax/income/subtractions/social_security/income_limit.yaml

```
@@ -0,0 +1,23 @@
+description: Connecticut qualifies filers for the social security benefit adjustment with adjusted gross income below this amount, based on filing status.
```

Your description does not accurately capture the logic as in the law, form, and Python code. Filers with AGI over the threshold are still eligible for a partial subtraction.
```suggestion
description: Connecticut subtracts all taxable Social Security benefits from state adjusted gross income for filers with adjusted gross income below this amount, based on filing status; otherwise it subtracts only part.
```